account_chart_update/model/account_tax_code.py

```diff
@@ -25,3 +25,13 @@ class AccountTaxCode(models.Model):
     _inherit = 'account.tax.code'
 
     active = fields.Boolean('Active', default=True)
+
+    def _sum(self, cr, uid, ids, name, args, context, where='',
+             where_params=()):
+        try:
+            return super(AccountTaxCode, self)._sum(
+                cr, uid, ids, name, args, context, where=where,
+                where_params=where_params)
+        except:
+            cr.rollback()
+            return dict.fromkeys(ids, 0.0)
```

Comment:
Rollback is needed because cr operation gives error and doesn't allow to continue. I want to ellaborate a patch, but now my priority is to finish the other tasks.